### ISSUE TITLE
VSCode Bug report: sourcegraph private instance doesn't work with vscode plugin

### ISSUE THREAD

**Issue by xenador1** [sourcegraph/sourcegraph#44227]
I'm using a privately hosted version of sourcegraph which is running version  4.1.1 when I generate an access token from my account https://sourcegraph.factset.com/users/jluchak/settings/tokens and use the copy button on the UI to obtain the token I get a failure to authenticate:

![image](https://user-images.githubusercontent.com/43618258/201180640-5ec476d1-bcef-45f7-918e-86a5d83b15f5.png)

Is this a bug in the plugin or is the server instance simply too old to support the use of the plugin?

**Comment by DepthDeluxe** [comment]
+1 on this issue, am seeing the same failure on my private instance of Sourcegraph too.

**Comment by xenador1** [comment]
I managed to get this working. Not sure which of these solved the issue but here's what happened since I last tried:

1. I waited for the server instance to upgrade to 5.1.2 (which I could see on the page: https://sourcegraph.factset.com/users/jluchak/settings/tokens, at the bottom of the list of links.)
2. I generated a new token from the same URL post-upgrade. This token looked to have a different format, at least one `_` near the front.
3. In vscode I uninstalled the plugin and reinstalled v2.2.16 of `Sourcegraph`.
4. In vscode I used the windows hotkeys `ctrl` + `,` to pull up the settings then in the User settings I used the icon to open the JSON file and added `"sourcegraph.url": "https://sourcegraph.factset.com",` in the middle of the settings then did `ctrl` + `shift` + p, and did `Developer: Reload Window`. This time vscode prompted me for permission so I left-clicked `Allow`. It then let me fill in my new token in the sidebar I showed previously and all was well.
